assets/monero.conf.template

```diff
@@ -1,41 +1,41 @@
+data-dir=BITMONERO_DIR          # Remember to create the monero user first
 
 # Log file
 log-level=0,blockchain:INFO
-log-file=/data/.bitmonero/monero.log
-max-log-file-size=10000000            # Currently 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
+log-file=MONERO_LOG
+max-log-file-size=10000000     # Our default: 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
 max-log-files=2
 
 # P2P full node
-p2p-bind-ip=0.0.0.0            # Bind to all interfaces (the default)
-p2p-bind-port=18080            # Bind to default port
+p2p-bind-ip=MONEROD_BIND_IP            # Bind to all interfaces (the default)
+p2p-bind-port=MONERO_P2P_PORT            # Bind to default port
 
-out-peers=ADV_P2P_MAXNUMOUTPEERS    # This will enable much faster sync and tx awareness; the default 8 is suboptimal nowadays [defacto default: 64]
-in-peers=ADV_P2P_MAXNUMINPEERS      # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
+out-peers=ADV_P2P_MAXNUMOUTPEERS         # Larger values will enable much faster sync and tx awareness
+in-peers=ADV_P2P_MAXNUMINPEERS          # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
 
-limit-rate-up=RATELIMIT_KBPSUP       # Default 2048 kB/s; contribute more to p2p network
-limit-rate-down=RATELIMIT_KBPSDOWN   # Default 8192 kB/s; allow for faster initial sync
+limit-rate-up=RATELIMIT_KBPSUP        # Default 2048 kB/s; contribute more to p2p network
+limit-rate-down=RATELIMIT_KBPSDOWN      # Default 8192 kB/s; allow for faster initial sync
 
 # RPC open node
-rpc-bind-ip=0.0.0.0            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
-rpc-bind-port=18081            # Bind on default port
+rpc-bind-ip=MONEROD_BIND_IP            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
+rpc-bind-port=MONERO_RPC_PORT            # Bind on default port
 confirm-external-bind=1        # Confirm we want the unrestricted RPC (set by rpc-bind-ip) listening on all network interfaces
                                # confirm-external-bind=1 and public-node=1 are required for node to be advertized in p2p handshake
 #restricted-rpc=1              # Prevent unsafe RPC calls (can be used to restrict the normally unrestricted RPC interface)
 rpc-access-control-origins=*
-rpc-restricted-bind-ip=0.0.0.0 # Bind restricted RPC to all interfaces
-rpc-restricted-bind-port=18089 # Bind restricted RPC on custom port to differentiate from default unrestricted RPC (18081)
+rpc-restricted-bind-ip=MONEROD_BIND_IP # Bind restricted RPC to all interfaces
+rpc-restricted-bind-port=MONERO_RPC_PORT_RESTRICTED # Bind restricted RPC on custom port to differentiate from default unrestricted RPC (18081)
 
 # Slow but reliable db writes
 db-sync-mode=safe
```

Comment:
```suggestion
#db-sync-mode=safe
```

increases wear

Reply:
What makes you say that?

Reply:
Its a fact. Safe performs more freq write cycles

Reply:
The docs say:
`If you are concerned with system crashes use safe:sync.`
We are definitely concerned with system crashes...